### DISCUSSION TITLE
Calling GetGraphic restarts the Linux container

### DISCUSSION THREAD

**Discussion by RonaldRijsdorp** [Shane32/QRCoder#426] [Q&A]
The following code runs fine in on my own windows laptop (application nor deployed in a container). When i deploy the application in a linux docker container the container  restarts after calling the GetGraphic method:

using var qrGenerator = new QRCodeGenerator();
 var qrCodeData = qrGenerator.CreateQrCode($"{url}?token={token}", QRCodeGenerator.ECCLevel.Q);
 using var qrCode = new Base64QRCode(qrCodeData);
 string qrCodeHtml = $"<img style='max-width:200px; width:100%' alt='Embedded Image' src=data:image/png;base64,{qrCode.GetGraphic(20)}/></td>";
         
In the container i added  libgdiplus and  libc6-dev

RUN apt-get install -y apt-utils libgdiplus libc6-dev

**Comment by RonaldRijsdorp** [comment]
In my issue description  the coding line that caused the issue was replaced by an embedded image link when i saved the issue. Coding was an HTML string with an img that and a the GetGraphic call : qrCode.GetGraphic(20)

**Reply by CssaBee** [reply]
Bitmap in System.Drawing.Common is only supported on Windows.
GetGraphic returns a Bitmap object, so it should throw a PlatformNotSupportedException.
https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/system-drawing-common-windows-only

**Reply by RonaldRijsdorp** [reply]
Thanks for your answer. I switched to ZXing.SkiaSharp.BarcodeWriter to solve this issue.
